client/view/workbook/select-sheet/SelectSheet.html

```diff
@@ -1,6 +1,6 @@
 <div class="container">
 	<div class="content">
-		<h1>{workbook.name}</h1>
+		<h1 class="center">{workbook.name}</h1>
 
 		{#if workbook.sheets.length === 1}
 			<p style="text-align: center">
```

Comment:
If we're using a class, may as well use it everywhere!
```suggestion
			<p class="center">
```